### ISSUE TITLE
Frontend-Backend Connection: Overview of digital identities

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#32]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Connection between database/backend and frontend is established.
* Correct data from database is displayed in the overview of the digital identities

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
> include these columns: ID, Name, Surname, Email, open credentials, open Proofs, Connection status

Some columns do not exists:
* open credentials
* open Proofs 
* Connection status

Shall I add dummy data? Currently, we do not store these informations and we do not have a database design which supports it.

**Comment by Idontker** [comment]
@TamaraHoock @veitmo 
see comment above. I forgot to tag you ^^"

**Comment by Idontker** [comment]
@TamaraHoock @veitmo
I am proposing to change "Connection status" to "Connection activated" in order to make it a boolean value (true/false)
